ios/FluentUI/People Picker/InitialsView.swift

```diff
@@ -12,7 +12,7 @@ import UIKit
  The initials are generated from a provided primary text (e.g. a name) or secondary text (e.g. an email address) and placed as a label above a colored background.
  */
 class InitialsView: UIView {
-    private static func initialsBackgroundColor(fromPrimaryText primaryText: String?, secondaryText: String?) -> UIColor {
+    static func initialsBackgroundColor(fromPrimaryText primaryText: String?, secondaryText: String?, colorOptions: [UIColor]? = nil) -> UIColor {
```

Comment:
this won't update the initials view if the token is changed?

Reply:
Nope. The AvatarVnext does not use the InitialsView.
We are just borrowing the functionality of the Initials calculation and the hash to determine the background color (which  will be used to calculate the ring and foreground colors shortly.